core/src/main/resources/reference.conf

```diff
@@ -30,6 +30,7 @@ avastRabbitMQConnectionDefaults {
     period = 5s
   }
 
+  useKluzo = false
```

Comment:
Have anyone used this feature? The removal would also simplify the code-base.

Reply:
Removed